lib/web_ui/dev/test_runner.dart

```diff
@@ -165,6 +168,16 @@ class TestCommand extends Command<bool> {
   /// Paths to targets to run, e.g. a single test.
   List<String> get targets => argResults.rest;
 
+  /// The target test files to run.
+  ///
+  /// The value can be null if the developer prefers to run all the tests.
```

Comment:
Is the result different if you didn't convert `isEmpty` to `null` and just let it be empty?

Reply:
the check which is added previous is for non-null, line 351